tests/src/Functional/DescriptionListFieldWidgetTest.php

```diff
@@ -49,6 +49,7 @@ protected function setUp() {
       'entity_type' => 'node',
       'field_name' => 'description_list',
       'bundle' => 'page',
+      'required' => TRUE,
```

Comment:
It is better also to have a field with an optional field.
Probably during execution, we can simply update the field config setting.
Everything else looks good.